rtl/include/eclhelper.hpp

```diff
@@ -48,7 +48,7 @@ typedef unsigned short UChar;
 
 //Should be incremented whenever the virtuals in the context or a helper are changed, so
 //that a work unit can't be rerun.  Try as hard as possible to retain compatibility.
-#define ACTIVITY_INTERFACE_VERSION      654
+#define ACTIVITY_INTERFACE_VERSION      655
```

Comment:
Is this change valuable enough to justify introducing a query incompatibility between 8.6 and 8.8 ? Do we need to wait for 9.x ?